tests/meta/transfer/test_external_data.py

```diff
@@ -0,0 +1,210 @@
+"""Tests for the transfer of external data"""
+import logging
+from pathlib import Path
+from typing import List
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.meta.transfer.external_data import ExternalDataAPI
+from cg.meta.transfer.md5sum import check_md5sum, extract_md5sum
+from cg.models.cg_config import CGConfig
+from cg.store import Store, models
+from tests.cli.workflow.conftest import dna_case
+from tests.mocks.hk_mock import MockHousekeeperAPI
+
+
+def test_create_log_dir(caplog, external_data_api: ExternalDataAPI, ticket_nr: int):
+    """Test generating the directory for logging"""
+    caplog.set_level(logging.INFO)
+
+    # WHEN the log directory is created
+    log_dir = external_data_api.create_log_dir(ticket_id=ticket_nr, dry_run=True)
+
+    # THEN the path is not created since it is a dry run
+    assert "Would have created path" in caplog.text
+
+    # THEN the created path is
+    assert str(log_dir).startswith("/another/path/123456")
+
+
+def test_create_source_path(
+    external_data_api: ExternalDataAPI, ticket_nr: int, customer_id: str, cust_sample_id: str
+):
+    """Test generating the source path"""
+
+    # WHEN the source path is created
+
+    source_path = external_data_api.create_source_path(
+        raw_path="/path/%s",
```

Comment:
```suggestion
        raw_path="/path/%s",
```
What does this one do?

Reply:
It's basically just a replacement for the hasta path gotten from the context config file. Do you want it to be a fixture?

Reply:
👍